### ISSUE TITLE
[Bug]: Thumbnail generation fails with image editable with cropping and cache buster

### ISSUE THREAD

**Issue by valentin-** [pimcore/pimcore#16439]
### Pimcore version

11.1.4

### Steps to reproduce

Hi, this project was updated from Pimcore 10 to Pimcore 11. There was no problem in Pimcore 10.

I've experienced an issue when using image editables **with** cropping and **cache buster active** in some cases. 

```
{{ pimcore_image('image', {
    thumbnail: 'teaser',
    cacheBuster: true,
}) }}
```

Example path frontend output:
```
/cache-buster-5/folder1/folder2/2147/image-thumb__2147__teaser_auto_8d1d280eddf50a7e0ad7b3d7a89b0635/filename@2x.aba9c031.jpg
```

In this case $thumbnailConfig is empty, which results in showing the fallback svg image (/bundles/pimcoreadmin/img/filetype-not-supported.svg)
https://github.com/pimcore/pimcore/blob/2b3bb4d10f4c32270589bb37557b932ca38d2a7e/models/Asset/Service.php#L509

After disabling the cachseBuster, the image shows up.

### Actual Behavior

-

### Expected Behavior

-

**Comment by fashxp** [comment]
could you please check if that fix would also fix your problem? Thx. 
https://github.com/pimcore/pimcore/pull/16467

**Comment by valentin-** [comment]
@fashxp I changed the two lines locally, but it didn't resolve the issue.

**Comment by fashxp** [comment]
@valentin- and now?

**Comment by valentin-** [comment]
@fashxp yes looks good now :+1:

**Comment by fashxp** [comment]
fixed via #16467
